### DISCUSSION TITLE
Hyper-parameter for the Random method

### DISCUSSION THREAD

**Discussion by kanzaali** [optuna/optuna#3867] [Q&A] [answered]
Hi,
I want to find the optimal hyperparameter for the Random method(Image-Classification) for the image dataset(CIFAR10, CIFAR100, ImageNet). The issue is when I train my model with OPTUNA the accuracy for CIFAR100 show only 50%, though without OPTUNA accuracy is higher. I wonder if I am reading the graph incorrectly or if my code has some issue. I have trained this model for so many days for 500 OPTUNA Trials       
![FailedOPTUNASliceRandom](https://user-images.githubusercontent.com/33715985/183794111-7f3cfdbf-db7e-4171-8907-650343dbf9e5.png)

**Comment by nzw0301** [comment, marked as answer]
According to the figure, the later trials have a much smaller accuracy than the ones of early trials, so I suppose that the code specifies a wrong direction for `study.optimize`: `direction=None` or `direction="minimize"`. Could you check whether or not to set `direction="maximize"`? 

Ref: https://optuna.readthedocs.io/en/stable/reference/generated/optuna.create_study.html

**Comment by kanzaali** [comment]
Thank you so much for your reply. I tried with create_study(direction="maximize") and attaching the output. how about increasing the accuracy? 
![SliceRandom](https://user-images.githubusercontent.com/33715985/184281611-cf19bfb2-e315-416c-b904-f58d85d12bd8.png)

**Reply by nzw0301** [reply]
Since we don't know the code you use, we cannot give a concrete answer. But as a general ways, I would say that 

- increasing the number of trials
- enlarging explored hyper-parameter space
- Use different model architecture / data augmentation
